docs/Tiered-Storage-on-Tachyon.md

```diff
@@ -103,6 +103,12 @@ Currently only synchronous eviction is supported by tiered storage. It is recomm
 small block size (less than 64MB), to reduce the latency of block eviction. This restriction will
 not exist when asynchronous eviction is introduced.
 
+## Space Reserver
+
+Space reserver makes tiered storage try to reserve certain portion of space on each storage layer
+before all space on some layer is consumed. It will be good for burst write performance, but for
```

Comment:
NIT: It will benefit the performance of bursty write, but may provide marginal performance gain for continuous writes when eviction becomes the bottleneck.